README.md

```diff
@@ -111,6 +111,14 @@ python setup.py install
         'METADATA_LOCAL_FILE_PATH': '[The metadata configuration file path]',
         'KEY_FILE': '[The key file path]',
         'CERT_FILE': '[The certificate file path]',
+        
```

Comment:
Please add these to the [module settings](https://github.com/grafana/django-saml2-auth?tab=readme-ov-file#module-settings) below as well.

Reply:
@mostafa I've updated the module settings!